cli/ioctl/cmd/account/accountimportkeystore.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package account
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+	"syscall"
+
+	"go.uber.org/zap"
+	"golang.org/x/crypto/ssh/terminal"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+
+	"github.com/iotexproject/iotex-core/cli/ioctl/cmd/config"
+	"github.com/iotexproject/iotex-core/cli/ioctl/validator"
+	"github.com/iotexproject/iotex-core/pkg/log"
+)
+
+// accountImportKeyStoreCmd represents the account import keystore command
+var accountImportKeyStoreCmd = &cobra.Command{
```

Comment:
Can we combine the command with import/export with private key string? For example,

import/export --keystorefile (--privatekey)